src/CMakeLists.txt

```diff
@@ -510,11 +526,11 @@ function(add_doom_tiny SUFFIX RENDER_LIB)
         USE_DIRECT_MIDI_LUMP=1
 
         NO_USE_NET=1 # standard networking
-        USE_PICO_NET=1
+        USE_PICO_NET=0
```

Comment:
i2c networking has been removed. There are two i2c controllers in an rp2040, so theoretically you could still run the network code independent of an i2c screen, I just haven't looked into it.